analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -62,6 +62,11 @@ import org.ossreviewtoolkit.utils.ort.HttpDownloadError
 import org.ossreviewtoolkit.utils.ort.OkHttpClientHelper
 import org.ossreviewtoolkit.utils.ort.showStackTrace
 
+/**
```

Comment:
Have you considered putting them into the companion object?

Reply:
Ah, right, that should be done to avoid name clashes. I'll add another commit.